rest-api/tools/auth_setup.sh

```diff
@@ -88,10 +88,40 @@ function get_instance_connection_name {
 }
 
 function get_db_password {
+  user=$1
   echo "Getting database password..."
   tools/install_config.sh --key db_config --instance $INSTANCE \
       --creds_file ${CREDS_FILE} --config_output $TMP_DB_INFO_FILE
-  PASSWORD=`grep db_password $TMP_DB_INFO_FILE | cut -d\" -f4`
+  if [ "$user" == "$RDR_DB_USER" ]
```

Comment:
You've got two if statements here with repeated ifs and complicated grep logic repeated... this is more complicated than what you had before...

Why not just:

if [ "$user" == "$RDR_DB_USER" ]
then
  PASSWORD_KEY = "db_password"
elif [ "$user" == $READONLY_DB_USER"]
then
  PASSWORD_KEY = "read_only_db_password"
...
PASSWORD =$(grep $PASSWORD_KEY...)

Reply:
ah, misunderstood your previous comment. Done.